### PR TITLE
Int 34 restrict individual screening access

### PR DESCRIPTION
**Pull request by AmanShurafa**
### Jira Story

- [Potential risk, any screenings not belonging to users can be access by backdoor 34](https://osi-cwds.atlassian.net/browse/INT-34)

## Note to reviewers
No 401 vs 403 discussion. 
418 teapot meme

### CODE REVIEW

File: spec/features/errors_spec.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@
         skip 'Pending as this test fails during parallel runs'
       end
       allow(Rails.configuration).to receive(:intake).and_return(dashboard_url: dashboard_url)
+      stub_request(:get, '/this_page_does_not_exist').and_return(json_body('NotFound', status: 500))
```
Comment (ThomasOSI) [inline spec/features/errors_spec.rb]: This should have status 404

File: app/javascript/errors/UnauthorizedPage.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import * as IntakeConfig from 'common/config'
+
+const UnauthorizedPage = () => (
+  <div className='error-panel centered'>
+    <h4>
+      <i className='fa fa-warning c-red gap-right'/>
+      Sorry, you are restricted from accessing this page.
```
Comment (brunoOSI) [inline app/javascript/errors/UnauthorizedPage.jsx]: Weird english here. Ask Daniel for a message.

File: spec/javascripts/components/errors/UnauthorizedPageSpec.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import UnauthorizedPage from 'errors/UnauthorizedPage'
+import {shallow} from 'enzyme'
+import * as IntakeConfig from 'common/config'
+
+describe('UnauthorizedPage', () => {
+  const dashboardUrl = 'my_dashboard_link'
+  let component
+
+  beforeEach(() => {
+    spyOn(IntakeConfig, 'dashboardUrl').and.returnValue(dashboardUrl)
```
Comment (brunoOSI) [inline spec/javascripts/components/errors/UnauthorizedPageSpec.jsx]: this whole dashboard thing should be intake's root path. that is the index page.

File: spec/features/errors_spec.rb [hunk 4]
```diff
@@ -29,6 +29,19 @@
     end
   end
 
+  context 'when user attempts to access a screening created by another' do
+    scenario 'renders 401 page' do
+      pending 'implemented but fails due to lack of js redirect support'
```
Comment (brunoOSI) [inline spec/features/errors_spec.rb]: Implemented but fails?

**Pull request closed without merging**